nodes/materials/emission.py

```diff
@@ -40,14 +41,19 @@ class LuxCoreNodeMatEmission(bpy.types.Node, LuxCoreNode):
     bl_width_default = 160
 
     emission_units = [ ("artistic", "Artistic", "Artist friendly unit using Gain and Exposure"),  
-        ("power", "Power", "Radiant flux in Watts")
+        ("power", "Power", "Radiant flux in Watt"),
+        ("lumen", "Lumen", "Luminous flux in Lumen"),
+        ("candela", "Candela", "Luminous intensity in Candela")
```

Comment:
Please add an integer for each enum entry, like so:
```python
("power", "Power", "Radiant flux in Watt", 0),
("lumen", "Lumen", "Luminous flux in Lumen", 1),
```
This is important so we have the option to reorder the elements at a later time, without breaking backwards compatibility.